benchmark/bench_CQRRP/CQRRP_runtime_breakdown.cc

```diff
@@ -48,7 +48,7 @@ static void data_regen(RandLAPACK::gen::mat_gen_info<T> m_info,
                                         QR_speed_benchmark_data<T> &all_data, 
                                         RandBLAS::RNGState<RNG> &state) {
 
-    RandLAPACK::gen::mat_gen<double, r123::Philox4x32>(m_info, all_data.A.data(), state);
+    RandLAPACK::gen::mat_gen<double>(m_info, all_data.A.data(), state);
```

Comment:
``double`` can be inferred from ``all_data.A.data()``.

Reply:
Fixed.